opencti-platform/opencti-front/src/private/components/analyses/external_references/ExternalReferenceStixCoreObjects.jsx

```diff
@@ -52,6 +52,7 @@ const ExternalReferenceStixCoreObjectsComponent = ({ externalReference }) => {
               </ListItemIcon>
               <ListItemText
                 primary={stixCoreObjectOrRelationship?.representative?.main}
+                primaryTypographyProps={{ style: { overflow: 'hidden', textOverflow: 'ellipsis' } }}
```

Comment:
```suggestion
                primaryTypographyProps={{ style: { wordWrap: 'break-word' } }}
```

Why this suggestion ? With `textOverflow: 'ellipsis'` we truncate the text and by so lose information. Using `wordWrap: 'break-word'` force the text to wrap to new line even if it is a very long word.

Reply:
Done, thanks for the suggestion :)